docs/build-system-migration.md

```
@@ -0,0 +1,151 @@
+       "build": "pnpm -r build"
+     },
+     "pnpm": {
+       "overrides": {
+         "@turf/difference": "6.0.1"
+       }
+     }
+   }
+   ```
+
+3. **Versioning/Publishing**
+   ```json:package.json
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)